modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -199,6 +199,7 @@ interface DOMUtils {
   dispatch: (target: Node | Window, name: string, evt?: {}) => EventUtils;
   getContentEditable: (node: Node) => string | null;
   getContentEditableParent: (node: Node) => string | null;
+  getContentEditableRoot: (node: Node) => HTMLElement | null;
```

Comment:
If you're adding a new public API then you **must** add JSDoc (moxiedoc) content so it shows in the API docs and you need a changelog entry as well.

Additionally the implementation below really does not make sense given this function name... the contenteditable root should be the one that is either `false` or `true` but your logic below only returns an element for `true`. It's also not likely going to be useful to as many if it only returns an element for a true context, so I'd suggest making it return an element in both cases and then change the code where it's being used to check if it's `true` or `false`.

- [ ] Rethink this API and make sure to add the appropriate docs/changelog entries if you keep it

Reply:
Will rework this to return the false element as well as you suggest, the original use can be changed so that the method is more suitable for public use. I plan to still return the first non-editable parent element, or furthest editable parent element, as this fits the original purpose the most.

Should I add the JSDoc content for `getContentEditable` and `getContentEditableParent` as well, since they are missing?